aws/ci_build.rake

```diff
@@ -234,7 +234,16 @@ end
 task :chef_update do
   if CDO.daemon && CDO.chef_managed
     RakeUtils.with_bundle_dir(cookbooks_dir) do
-      RakeUtils.bundle_exec 'berks', 'install'
+      # Automatically update Chef cookbook versions in staging environment.
+      if rack_env?(:staging)
+        RakeUtils.bundle_exec './update_cookbook_versions'
+        RakeUtils.bundle_exec 'berks', 'install'
```

Comment:
Will running `berks install` here after the `./update_cookbook_versions` script update our root Berksfile.lock, or do we still need to do that manually?